package.json

```diff
@@ -31,6 +31,7 @@
     "@typescript-eslint/parser": "^3.6.1",
     "autocannon": "^6.0.0",
     "fastify": "^3.0.2",
+    "graphql-constraint-directive": "^2.1.0",
```

Comment:
Question: Is it necessary to pull in a new 3rd party package to test the functionality?